infra/k8s/helm/Titus/.helmignore

```diff
@@ -0,0 +1,23 @@
+# Patterns to ignore when building packages.
```

Comment:
this comment is about file naming: do we need the Titus folder in the directory structure? We are in the titus repository, so this can only be about Titus. Let's remove it unless there are reasons to keep it there

Reply:
We do not really need it